Monika After Story/game/script-topics.rpy

```diff
@@ -17093,3 +17093,46 @@ label monika_tanabata:
     else:
         m 3eua "Even if you're not interested in travelling, it's pretty interesting to explore other cultures, isn't it?"
     return
+
+#NOTE: Unlocked after Monika reacts to reddit (I hope)
```

Comment:
If you want this topic to be shown after having seen the Reddit react, you should add a `conditional` property in `Event`, ideally you'd be checking for `"renpy.seen_label('mas_wrs_reddit')"`, and linked to an `action` of `EV_ACT_RANDOM`